docs/guide/data.md

```diff
@@ -169,3 +174,11 @@ time,tair,rh,slp,precip
 ...
 2019-07-01T09:00:00+0000,74.3,97.0,1019.0,0.02
 ```
+
+## Table Timezones
+
+Although most tables store data in UTC time, there are a few notable exceptions when UTC data is meaningless or is more confusing so the data is stored or returned in local time so the `local_time` parameter is not needed:
+
+- The `daily` table is stored in local time. As this table contains aggregations, averages, and extremes from a day, it's most meaningful if that is the local 00:00 to 23:59 day and not the UTC 00:00 to 23:59 day.
+
+- The `local_hourly` table is stored in UTC time but is offset by the location timezone *without* needing the `local_time` parameter. As not all timezones fall evenly on the hour (e.g. UTC+9:30), this table converts the UTC time so that you always have timestamps that fall on the hour (e.g. 00:00 or 01:00) even in cases where the offset is not a full hour.
```

Comment:
`local_time` parameter is also needed for `local_hourly` table, if users need local timestamps in the response

Reply:
Ah, I was basing it off the description that was in the Jira task: "The local_hourly table is in UTC but offset by the location timezone."

Does this just mean it's aggregated so that the UTC + offset will fall on the hour?

Reply:
Right, for example in 9.5 timezone, hourly table has timestamp in 1:00, local_hourly has timestamp in 1:30 which considers the 30 mins offset

Reply:
Ok, removed that from the table timezones section so it's just the note that daily data is stored in local time, not UTC.